internal/payloads/reservation_payload.go

```diff
@@ -28,6 +28,8 @@ type AWSReservationRequest struct {
 		NewName    *string `json:"new_name"`
 		NewBody    *string `json:"new_body"`
 	}
+	// We have pubkey here as a struct and in the AWSReservation
+	*models.AWSReservation
```

Comment:
This will expose the whole reservation which is fine, but we need to ensure that users are unable to set things like created_at, state or finished_at. Since the Create DAO method does not allow setting them explicitly, it is probably fine since there is no update endpoint, but I think it is worth documenting this in the OpenAPI (that some fields must not be set).